cardano-crypto-class/src/Cardano/Crypto/KES/Simple.hs

```diff
@@ -12,6 +12,9 @@
 {-# LANGUAGE NoStarIsType #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 
+-- Needed for ghc-9.6.
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+
```

Comment:
Why is this needed only for 9.6? If we remove the redundant constraints it will fail with 9.2? 8.10?

Reply:
Nope. Only `ghc-9.6` needs it.